### DISCUSSION TITLE
Commandpost slow?

### DISCUSSION THREAD

**Discussion by Tegels038** [CommandPost/CommandPost#3167] [General]
I'm working with FCPX (10.6.5) on a Imac (with ventura 13.0.1), 3,6 GHz 8-core Intel Core i9 with 72 GB DDR4 memory.
When working within FCPX, I do my color correction with the dials on my Loupedeck+. I made a selection. The dials are well selected but the response time is so slow and jerky/choppy... What can cause this problem and, more important: how can I solve this problem?
Commandpost is also the lastest version 1.4.14

**Comment by latenitefilms** [comment]
Are there any errors in the debug console? Is the official Loupedeck software not running?

**Reply by Tegels038** [reply]
Thnx for your response! The official Loupedeck software isn't installed on my iMac, just Commandpost. Ik made a printscreen of the debug console. There is some info about 'slow plugin'.... Is this normal or not? Solution?
<img width="491" alt="Scherm­afbeelding 2022-12-09 om 12 54 10" src="https://user-images.githubusercontent.com/120023865/206697414-f5f33316-e20e-4381-b809-ab162b1e6e95.png">

**Reply by latenitefilms** [reply]
Those slow plugin start times are unrelated. Can you share a screenshot of the Loupedeck+ preferences panel for one of the knobs that is slow? What actions are you using? Are there any errors in the debug console when you turn the knobs and it's slow to react? Is your Loupedeck+ connected directly to your Mac (as opposed to USB hub)? If you disable MIDI support in CommandPost, does it have any impact?

**Reply by Tegels038** [reply]
Here's a screenshot of the Highlights dial. It controls the Highlights (brightness) in the Color Wheels. When turning the dial, I don't see errors in the debug console. The loupedeck+ is now mounted directly to the USB port of the iMac (before not, via hub but I cannot see any difference). MIDI support disabled, no difference.
<img width="986" alt="Scherm­afbeelding 2022-12-09 om 13 09 36" src="https://user-images.githubusercontent.com/120023865/206701004-a2af7cae-d046-4884-ace5-b5d7b0c01e4a.png">

**Reply by latenitefilms** [reply]
Have you tried trashing FCPX's preferences? If you're using dual screens, can you try using a single screen in the default layout?

I haven't tried the latest Ventura update yet - so I'll try and reproduce your exact setup when I can.

**Reply by Tegels038** [reply]
I have reset the preferences of FCPX, no result. I am not working with dual screens.

**Comment by rickdhoy** [comment]
Hi, I am also having a great Deal of issues after installing CommandPost.

We have been using FCPX for many, many years (1999) with out any real issues ( other than the normal ) with Loupdeck but since the advice from Loupedeck to install Commandpost our little production company has come to a grinding halt. 4 Different Macs all running LoupdeckCTs and all of them have slowed down so significantly they are nearly useless. 

All machines running 

MAC OSX SONOMA 14.3.1
Loupdeck 5.8.1.18057
CommandPost 1.4.26
FCPX 10.7.1

All running on:

Apple M2 Ultra with 24-core CPU, 76-core GPU, 32‑core Neural Engine
192GB memory


We need to find a solution or start again !!


Clever Dogs

**Reply by latenitefilms** [reply]
CommandPost won't do anything if it's not running, so an easy way to rule out CommandPost being an issue is to just not run it, and see if you're still having issues.

Are you using the official Loupedeck software, or are you using CommandPost's Direct/Native Loupedeck support?

**Comment by matrix-compute** [comment]
ef0d42241c1d206c97d80d434c01654ef2bb8e79
